odoo/addons/base/models/ir_http.py

```diff
@@ -396,6 +396,10 @@ def _binary_record_content(
         if not content:
             if model == 'ir.attachment':
                 content = record.raw
+            elif (field_def.related_field and
+                  field_def.related_field.name == 'raw' and
+                  field_def.related_field.model_name == 'ir.attachment'):
```

Comment:
lint: I would have the closing parenthesis on a new line, with same indent level as the line with the open parenthesis

And inside content would be indented 4 spaces, not visually aligned.

Or maybe just put everything on a single line.